superset/Dockerfile

```diff
@@ -87,11 +87,16 @@ RUN microdnf update \
         openldap-clients \
         openssl-libs \
         openssl-pkcs11 \
+        tar \
         python${PYTHON//./} \
     && microdnf clean all
 
 COPY superset/licenses /licenses
-COPY --from=statsd-exporter /bin/statsd_exporter /stackable/statsd_exporter
+
+# Choosing correct binary with respect to architecture
+RUN ARCH=$(arch | sed 's/^aarch64$/arm64/' | sed 's/^x86_64$/amd64/') \
+    && pushd /stackable && curl --fail -L https://repo.stackable.tech/repository/packages/statsd_exporter/statsd_exporter-${STATSD_EXPORTER}.linux-${ARCH}.tar.gz | tar -xzC /stackable && ln -s statsd_exporter-${STATSD_EXPORTER}.linux-${ARCH}/statsd_exporter statsd_exporter && popd
+
```

Comment:
Not a fan of push and pop d. Can we just use "curl -o .." or workdir or at least use wrapping (" && \") for better readability?